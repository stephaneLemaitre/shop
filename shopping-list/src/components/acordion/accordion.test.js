import React from 'react';
import Accordion from "./accordion";
import renderer from 'react-test-renderer';
import {mockData} from "./mockdata";

it('it match snapshot', () => {
    const {categories} = mockData;
    const tree = renderer
        .create(<Accordion data={categories}/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});