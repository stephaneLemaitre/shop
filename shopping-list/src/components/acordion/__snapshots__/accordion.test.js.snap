// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it match snapshot 1`] = `
<div
  className="accordion"
>
  <div
    className="panel "
  >
    <div
      className="panel-content"
    >
      <span
        className="icon-cross"
        onClick={[Function]}
      />
      <div
        className="accordion"
      />
    </div>
  </div>
</div>
`;
